src/Miunie.Storage/JsonDataStorage.cs

```diff
@@ -76,39 +70,24 @@ public void WipeData()
             Directory.Delete(_resourcesFolder);
         }
 
-        public bool KeyExists(string collection, string key)
-        {
-            var file = GetFileNameByKey(key);
-            return LocalFileExists(String.Concat(collection, "/", file));
-        }
+        private string KeyToFullFileName(string key) => String.Format(FileTemplate, key);
 
-        private bool LocalFileExists(string file)
-        {
-            string filePath = String.Concat(_resourcesFolder, "/", file);
-            return File.Exists(filePath);
-        }
+        private string GetFullFilePath(string collection, string key) =>
+            Path.Combine(_resourcesFolder, collection, KeyToFullFileName(key));
 
-        private string GetFileNameByKey(string key)
-            => String.Format(FileTemplate, key);
+        private string GetCollectionDirectory(string collection) => Path.Combine(_resourcesFolder, collection);

```

Comment:
In this case, splitting the body into a new line, would also fit the 80 characters line limit rule.